src/btree/bt_sync.c

```diff
@@ -650,7 +650,11 @@ __wt_sync_file(WT_SESSION_IMPL *session, WT_CACHE_OP syncop)
 
                 /* Periodically log checkpoint progress. */
                 if (conn->ckpt_write_pages % 5000 == 0)
-                    __wt_checkpoint_progress(session, false);
+                    __wt_verbose_progress(session, WT_VERB_CHECKPOINT_PROGRESS,
+                      conn->ckpt_timer_start, &conn->ckpt_progress_msg_count, true,
+                      "Checkpoint has been running for %" PRIu64 " seconds and wrote: %" PRIu64
+                      " pages (%" PRIu64 " MB)",
+                      time_diff, conn->ckpt_write_pages, conn->ckpt_write_bytes / WT_MEGABYTE);
```

Comment:
Ah! did not notice before, referring to `time_diff` here where it is not defined. It is actually defined and used in the macro itself. That doesn't look clean. If we are going to use a common macro, maybe the message provided through the invocation of the macro can be appended to a generic message inside the macro. The message inside the macro could talk about the operation that has been running for x duration. I am not sure how hard that would be without changing the existing message we print for checkpoint duration. I am also not sure if it is okay to change the words for an existing message.

Reply:
I thought more about if there is a clean way of doing this using a macro, and without changing the message we already have for the checkpoint, it seems overly complicated. We should evaluate if we can change the wording of the checkpoint message, if not maybe switch to the other branch you created?

Reply:
Well the messages provided were intended to be use-case specific I don't think we can generalize them between the two if that was the idea?